en/Live preview update.md

```diff
@@ -1,61 +1,59 @@
-Live Preview is introduced to help you preview what you're writing on the go, as well as hide Markdown formatting.
+Obsidian 13 introduces a brand new editor engine that comes with several enhancements to your editing experience:
 
-See Live Preview in action:
+- Live Preview
+- Spellchecker
+- Text substitution
```

Comment:
this is a macOS service that used to not work on the editor, I don't know we want to sell it as an enhancement, could be misleading to windows and linux as they might not have native features like this